.github/workflows/ibis-docs-lint.yml

```diff
@@ -234,6 +260,9 @@ jobs:
           nix develop --ignore-environment -c \
             mkdocs gh-deploy --message 'docs: ibis@${{ github.sha }}' --ignore-version
 
+      - name: build and push quarto docs
+        run: nix develop --ignore-environment -c just docs-deploy
```

Comment:
This will fail to run due to

1. azure-cli not existing in the environment
2. no azure credentials existing

Reply:
Addressed. `az` is installed on GHA by default but doesn't build with the latest nixpkgs. I've put up a PR (https://github.com/NixOS/nixpkgs/pull/251560) to address the latter.

Reply:
In the meantime we can use the GHA installed version.